ingestion/src/metadata/profiler/metrics/system/system.py

```diff
@@ -483,6 +487,18 @@ def is_system_metrics(cls) -> bool:
     def name(cls):
         return "system"
 
+    def _manage_cache(self, max_size_in_bytes: int = 2147483648) -> None:
```

Comment:
nit - instead of `2147483648` could we have a variable named `2GB`? or show directly the computations 1024 * 1024...